ssl/ssl_locl.h

```diff
@@ -2034,6 +2034,17 @@ typedef enum downgrade_en {
 
 #define TLSEXT_SIGALG_ed25519                                   0x0807
 #define TLSEXT_SIGALG_ed448                                     0x0808
+#define TLSEXT_SIGALG_ecdsa_brainpoolP256r1_sha256              0x081a
+#define TLSEXT_SIGALG_ecdsa_brainpoolP384r1_sha384              0x081b
+#define TLSEXT_SIGALG_ecdsa_brainpoolP512r1_sha512              0x081c
+
+/* EC curve group ids */
+#define TLSEXT_GROUPID_brainpoolP256r1                          26
+#define TLSEXT_GROUPID_brainpoolP384r1                          27
+#define TLSEXT_GROUPID_brainpoolP512r1                          28
+#define TLSEXT_GROUPID_brainpoolP256r1_tls13                    31
+#define TLSEXT_GROUPID_brainpoolP384r1_tls13                    32
+#define TLSEXT_GROUPID_brainpoolP512r1_tls13                    33
```

Comment:
Maybe we want to add defines for things like x25519 too, and then update eccurves_default[] to use the defines instead of just the numbers.

Reply:
yes. there are two code points for P-256 and P-384

```
include/openssl/tls1.h:# define TLSEXT_curve_P_256                              23
include/openssl/tls1.h:# define TLSEXT_curve_P_384                              24
```

I was unsure why those are in a public API. Those have to stay, but
I would replace them with internal TLSEXT_GROUPID defines as needed.
Should I do that in this PR?